controller/documentcontroller.php

```diff
@@ -547,6 +547,9 @@ public function wopiCheckFileInfo($fileId){
 			return false;
 		}
 
+		$instanceId = \OCP\Config::getSystemValue('instanceid', '');
+		$instanceId = substr(md5($instanceId), -13); // Don't expose the secret ID.
```

Comment:
"secret" here refers to the instanceId. I assumed an admin wouldn't want to expose it, hence the md5.